SU-SWS/decanter
# Update TW to v3.4.1 and nvmrc to 20

# READY FOR REVIEW

# Summary
- Update nvmrc to 20
- Update TW to 3.4.1
- Other misc updates

# Needed By (Date)
- 🤷🏼‍♀️ 

# Urgency
- Not super urgent

# Steps to Test

1. Look at code to see what's changed
2. Look at preview and see that it's still working
3. Look at the Campaign preview in this PR that's using this branch of decanter and see that it works ok
https://github.com/SU-SWS/ood-giving-campaign/pull/189
https://deploy-preview-189--giving-campaign.netlify.app/
